addons/website_jitsi/data/website_jitsi_data.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <!-- Address of the Jitsi server to use -->
+        <record id="website_jitsi.jitsi_server_domain" model="ir.config_parameter">
+            <field name="key">website_jitsi.jitsi_server_domain</field>
+            <field name="value">meet.jit.si</field>
+        </record>
+    </data>
+</odoo>
```

Comment:
Do we really need that "data" if we have a clean fallback in the code?

Reply:
@awa-odoo Otherwise the setting will not be displayed in the "system parameter" view, and people might think that they can not change the domain name :/

Reply:
Ok I see your point, we will see what the reviewer thinks.

Reply:
Reviewer thinks this should be in a settings (probably website settings). I would put it inside Features section of Website settings, below google maps. No checkbox, just the right pane with "Jitsi", and the field with the config parameter (meet.jit.si placeholder : default value).